src/test/java/uk/gov/companieshouse/api/strikeoffobjections/chips/ChipsClientTest.java

```diff
@@ -35,16 +45,23 @@
     private RestTemplate restTemplate;
 
     @Test
-    void testSendMessageToChips(){
+    void testSendMessageToChips() {
+        Utils.setTestAttachmentsWithLinks(ATTACHMENTS);
         ReflectionTestUtils.setField(chipsClient, "chipsRestUrl", CHIPS_REST_URL);
         ChipsRequest chipsRequest = new ChipsRequest(
-                REQUEST_ID,
-                COMPANY_NUMBER
+                OBJECTION_ID,
+                COMPANY_NUMBER,
+                ATTACHMENTS,
+                OBJECTION_ID,
+                CUSTOMER_EMAIL,
+                REASON,
+                DOWNLOAD_URL_PREFIX
         );
 
         when(restTemplate.postForEntity(CHIPS_REST_URL, chipsRequest, String.class)).thenReturn(new ResponseEntity<>(HttpStatus.OK));
 
-        chipsClient.sendToChips(REQUEST_ID, chipsRequest);
+        chipsClient.sendToChips(OBJECTION_ID, chipsRequest);
```

Comment:
first param needs to be the http request id

Reply:
Made sure request and objection ids are in the right places.